test/parallel/test-eventtarget-whatwg-once.js

```diff
@@ -1,4 +1,5 @@
 // Flags: --expose-internals --no-warnings
+
```

Comment:
My understanding is that we don't need this anymore, correct?
```suggestion
```

Reply:
I thought so but the other PR just merged this line in so i don't know 🤷‍♂️

Reply:
I've opened https://github.com/nodejs/node/pull/36002 to address this.

Reply:
okay great i'll remove them again from this PR

Reply:
No, it's OK, you can leave them as is, otherwise I'll get conflicts on the other PR 😅